virttest/shared/cfg/guest-os/Windows.cfg

```diff
@@ -39,7 +39,7 @@
     check_guest_bsod = yes
 
     # Set verifier flags for windows guests
-    Win10, Win2016, Win2019:
+    Win2016, Win2019:
```

Comment:
So skip Win10 for this checking directly? I saw you give win10 another value previously, why remove it now?

Reply:
There is default value for Win10 the same as win2012 guests in virttest/utils_test/qemu/__init__.py , so no need to add it separately.
145         if win_verifier_flags:
146             verifier_setup_cmd = "verifier /flags %s /driver %s.sys" % (
147                                  win_verifier_flags, ".sys ".join(driver.split()))
148         else:
149             verifier_setup_cmd = "verifier /standard /driver %s.sys" % (
150                                  ".sys ".join(driver.split()))

Reply:
@kimi1978wy I see, thanks for the declaration. LGTM now.